predicators/third_party/fast_downward_translator/axiom_rules.py

```diff
@@ -9,7 +9,6 @@
 
 
 class AxiomDependencies(object):
-
```

Comment:
looks like the autoformatting went wild here -- can you revert all these third party files?